api/test/integration/admin-regions/admin-regions-get-descendants.spec.ts

```diff
@@ -0,0 +1,150 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import { AppModule } from 'app.module';
+import { AdminRegion } from 'modules/admin-regions/admin-region.entity';
+import { AdminRegionsModule } from 'modules/admin-regions/admin-regions.module';
+import { AdminRegionRepository } from 'modules/admin-regions/admin-region.repository';
+import { createAdminRegion } from '../../entity-mocks';
+import { getApp } from '../../utils/getApp';
+import { AdminRegionsService } from 'modules/admin-regions/admin-regions.service';
+
+describe('AdminRegions - Get descendants by Admin Region Ids', () => {
+  let app: INestApplication;
+  let adminRegionRepository: AdminRegionRepository;
+  let adminRegionService: AdminRegionsService;
+
+  beforeAll(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [AppModule, AdminRegionsModule],
+    }).compile();
+
+    adminRegionRepository = moduleFixture.get<AdminRegionRepository>(
+      AdminRegionRepository,
+    );
+    adminRegionService = moduleFixture.get(AdminRegionsService);
+
+    app = getApp(moduleFixture);
+    await app.init();
```

Comment:
Improvement tip:

When we are not actually requiring the API to build a testing instance (like in this case, we are not sending requests and receiving responses over HTTP, there is no need to 
```
await app.init()
```

Even tho there is 'no harm' doing this, it's wasting resources and adding more runtime to the tests pipeline, since our pipeline is already quite heavy, we should avoid doing this as much as we can :)